runtime/fake_dawn.py

```diff
@@ -43,13 +43,14 @@ def receiver(port, receive_queue):
         runtime_message = runtime_pb2.RuntimeData()
         runtime_message.ParseFromString(msg)
         receive_queue[0] = msg
-        print(runtime_message)
+        #print(runtime_message)
```

Comment:
Should remove this line or keep it. Shouldn't just keep it commented out on master